>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_1.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_2.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_3.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_4.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_5.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_6.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_7.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_8.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_9.h5
>loaded ../../trained_models/models_segments_no_overlap_65%_10epochs/model_10.h5
Loaded 10 models
[[0.414 ]
 [0.414 ]
 [0.4132]
 [0.4168]
 [0.414 ]
 [0.414 ]
 [0.4324]
 [0.4332]
 [0.4332]
 [0.4324]
 [0.432 ]
 [0.434 ]
 [0.4304]
 [0.5064]
 [0.468 ]
 [0.468 ]
 [0.482 ]
 [0.494 ]
 [0.494 ]
 [0.4932]]
[[0.414]
 [0.414]
 [0.414]
 [0.414]
 [0.414]
 [0.414]
 [0.432]
 [0.432]
 [0.432]
 [0.432]
 [0.432]
 [0.432]
 [0.432]
 [0.468]
 [0.468]
 [0.468]
 [0.468]
 [0.468]
 [0.468]
 [0.468]]
['0.414', '0.414', '0.4132', '0.4168', '0.414', '0.414', '0.4324', '0.4332', '0.4332', '0.4324', '0.432', '0.434', '0.4304', '0.5064', '0.468', '0.468', '0.482', '0.494', '0.494', '0.4932']
wandb: ERROR plotly is required to log interactive plots, install with: `pip install plotly` or convert the plot to an image with `wandb.Image(plt)`
Traceback (most recent call last):
  File "C:\MyDocuments\Dizertatie\segments\wandb_visualization\no_overlap\test_mode_visualization.py", line 74, in <module>
    wandb_config.log({"plot" : plt})
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 393, in wrapper
    return func(self, *args, **kwargs)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 344, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 334, in wrapper
    return func(self, *args, **kwargs)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 1750, in log
    self._log(data=data, step=step, commit=commit)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 1531, in _log
    self._partial_history_callback(data, step, commit)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\wandb_run.py", line 1401, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\interface\interface.py", line 567, in publish_partial_history
    data = history_dict_to_json(run, data, step=user_step, ignore_copy_err=True)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\data_types\utils.py", line 52, in history_dict_to_json
    payload[key] = val_to_json(
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\data_types\utils.py", line 80, in val_to_json
    val = Plotly.make_plot_media(val)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\sdk\data_types\plotly.py", line 48, in make_plot_media
    val = util.matplotlib_to_plotly(val)
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\util.py", line 500, in matplotlib_to_plotly
    tools = get_module(
  File "C:\MyDocuments\Dizertatie\segments\venv\lib\site-packages\wandb\util.py", line 243, in get_module
    raise wandb.Error(required)
wandb.errors.Error: plotly is required to log interactive plots, install with: `pip install plotly` or convert the plot to an image with `wandb.Image(plt)`